docker/calamari.Dockerfile

```diff
@@ -32,8 +32,8 @@ RUN $PARA_BINARY_PATH --version
 ADD $PARA_GENESIS_URL $PARA_GENESIS_PATH
 ADD $RELAY_GENESIS_URL $RELAY_GENESIS_PATH
 
-# Expose three ports by default
-EXPOSE 30333 9933 9944
+# Expose 5 ports by default
+EXPOSE 30333 30334 9933 9944 9615
```

Comment:
are you sure the ws/rpc ports should be exposed by default?
I'm thinking interacting with them in this way is a dev usecase and devs can just invoke it with
`docker run -p 9933:9933 imagename`

Reply:
turns out i misunderstood the meaning of EXPOSE

> Exposing ports is a way of documenting which ports are used, but does not actually map or open any ports. Exposing ports is optional.
](https://docs.docker.com/engine/reference/builder/#expose)

resolving this

Reply:
Yeah, you still have explicitly to open the port while run a container even though the port is exposed in dockerfile.